CHANGELOG.md

```diff
@@ -11,12 +15,11 @@ Little things:
 
 - The transaction review page now shows the "From" line before the "To" line.
 - The recovery page instructions now indicate that you can enter a 24-word mnemonic.
-- We no longer show the `oasis1...` address for Ethereum-compatible accounts.
 - Creating new wallets now generates 24-word mnemonic.
```

Comment:
(Not sure if this should be promoted from "Little")

Reply:
it's the reason we're making this release 💀

Reply:
:sweat_smile:

Reply:
changed